### DISCUSSION TITLE
Not showing new events in Calendar

### DISCUSSION THREAD

**Discussion by bbasselgia** [MMRIZE/MMM-CalendarExt3#132] [Q&A] [answered]
Since installing the latest updates, I've noticed that the calendar doesn't show new events on my calendar while MagicMirror is running.   I've added some events to my Google Calendar, but they don't show up in MagicMirror.   If I stop MagicMirror and restart it, they show up.   But, new events don't seem to be getting picked up while MagicMirror is running.

**Comment by eouia** [comment]
To make things clear,
1) In the default calendar module of MM (not this CX3 module), Will the event be shown?

I think you may have some confusion about MM. When you add a new event to your Google Calendar, the calendar module doesn't reflect its change in real time. It will take some time (`updateInterval`) to fetch updated events. Are you sure your case is not that?

**Comment by bbasselgia** [comment]
I'll have to double-check that.   I have had the MM default calendar module hidden since I started using CX3.   I'll update my config file and do some testing.   I don't expect the events to show up instantly.   But, it's gone days without the new events showing up.

**Reply by bbasselgia** [reply, marked as answer]
Ok, I did some testing.   This was not a CX3 issue, sorry, I'm still kind of new at this.
FYI, the MM sample config file, has fetchInterval: 7 * 24 * 60 * 60 * 1000,  This is supposed to be the time in miliseconds between refresh.  I updated the fetch interval to 300000 milliseconds (5 Minutes).   The default calendar now updates every 5 minutes and CX3 updates about 5 minutes after that.   Which is fine for me.

Again, sorry for posting this in the wrong place.
